reflection/grpc_testing/dynamic.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
```

Comment:
Move this file to a new folder `reflection/testdata`, and make a `testdata` package like this https://github.com/grpc/grpc-go/blob/master/testdata/testdata.go